axelor-human-resource/src/main/java/com/axelor/apps/hr/service/employee/EmployeeFetchServiceImpl.java

```diff
@@ -74,4 +80,17 @@ protected String getFilters(HrBatch hrBatch) {
 
     return Joiner.on(" AND ").join(query);
   }
+
+  @Override
+  public List<Employee> getInvitedCollaborators(LocalDate expenseDate) {
+    return employeeRepository
+        .all()
+        .filter(
+            "self.user.blocked = false AND self.hireDate <= :expenseDate AND (self.leavingDate=null OR self.leavingDate >= :expenseDate) AND (self.user.expiresOn is null OR self.user.expiresOn> :currentDate) \n"
+                + "AND self.mainEmploymentContract.payCompany IN :companySet")
```

Comment:
- Please to improve readability, add a line return before each ` AND`
- Remove the `\n`